src/view/dataElements/xdmObject/helpers/getNodeEditData.js

```diff
@@ -75,14 +77,14 @@ const getNodeEditData = ({
         candidateFieldName: candidateFieldName
           ? `${candidateFieldName}.properties.${propertyName}`
           : `properties.${propertyName}`,
-        candidateDisplayName:
-          propertySchema.candidateDisplayName || propertyName
```

Comment:
`propertySchema.candidateDisplayName` never was a thing. I think this was a mistake I made when I built the XDM Object view. I think what I meant was `candidateDisplayName: propertySchema.title || propertyName`. I fixed this, which caused the breadrumb to use the friendly display names (when available) and not the property names, and I thought it was a little awkward from a user perspective to have the text in the breadcrumb be inconsistent with the node labels shown in the schema tree.

So......I changed this code, so that we're using the "property name" in the breadcrumb, which matches what's shown in the schema tree. We now render the friendly display name as a heading on the edit form below the breadcrumb. I think it makes for a pretty good UX, but I'm open to feedback.

Reply:
I like 👍🏻